docs/content/guide/theming.md

```diff
@@ -6,7 +6,7 @@ weight: -6
 # Theming
 
 Druxt components can be themed using two primary methods:
-- [Druxt wrapper components](#druxtwrapper)
+- [DruxtWrapper components](#druxtwrapper)
```

Comment:
I think `Druxt wrapper` component makes more sense, as it's a Wrapper component provided by Druxt.